operations/iuf/IUF.md

```diff
@@ -190,6 +191,7 @@ The `iuf` command-line interface is used to invoke all IUF operations. The `iuf`
 | activity        | Display IUF activity details, annotate IUF activity      |
 | list-activities | List all activities present on the system                |
 | list-stages     | Display stages and status for a given IUF activity       |
+| workflow        | List workflows or information for a particular workflow  |
 
 ### Global arguments
```

Comment:
Since this is self-documented via the app's own usage, should users just be directed towards `--help` instead?

Question relates to this PR but doesn't at all need to prevent this PR from merging.

Reply:
Good point  -- I'll ask about it.  I don't think it hurts,  but it means we need to remember to update it in two places.